.github/ISSUE_TEMPLATE/release.md

```diff
@@ -7,97 +7,108 @@ labels: release
 <!--
 - [ ] Created the [release branch][release-branch].
-- [ ] Created and merged an empty commit to the `master` branch
-  ([PR][empty-commit-pr]).
-- [ ] Run the [Tag workflow][rc-tag-workflow] on the `master` branch with the
-  next release candidate tag ([Tag][rc-tag]).
+- [ ] Created and merged an empty commit to the `master` branch ([PR][empty-commit-pr]).
+- [ ] Run the [Tag workflow][rc-tag-workflow] on the `master` branch with the next release candidate tag ([Tag][rc-tag]).
 -->
-- [ ] Updated all version information in the documentation on the relevant
-  release branch ([PR][docs-update-pr]).
-- [ ] Run the [Tag workflow][tag-workflow] on the relevant release branch with
-  the proper release version ([Tag][tag]).
-- [ ] Run the [CI workflow][ci-workflow] on the release branch and verified that
-  the tagged build version exists on the [releases.crossplane.io] `build`
-  channel. ([build][release-build]).
-- [ ] Run the [Configurations workflow][configurations-workflow] on the release
-  branch and verified  that version exists on [registry.upbound.io] for all
-  getting started packages.
-    - [ ] [xp/getting-started-with-aws]
-    - [ ] [xp/getting-started-with-aws-with-vpc]
-    - [ ] [xp/getting-started-with-azure]
-    - [ ] [xp/getting-started-with-gcp]
-- [ ] Run the [Promote workflow][promote-workflow] with channel `stable` on the
-  release branch and verified that the tagged build version exists on the
-  [releases.crossplane.io] `stable` channel ([build][stable-build]).
-- [ ] Published a [new release] for the tagged version, with the same name as
-  the version and descriptive release notes ([release][release]).
-- [ ] Updated the current release version in the [Crossplane docs website repo]
-  ([PR][website-pr]).
-- [ ] Updated the [releases table] in the `README.md` on `master`
-  ([PR][release-table-pr]).
-- [ ] Ensured that users have been notified of the release on all communication
-  channels:
-    - [ ] [Slack][slack]
-    - [ ] [Twitter][twitter]
+- [ ] Updated all version information in the documentation on the relevant release branch ([PR][docs-update-pr]).
+- [ ] Run the [Tag workflow][tag-workflow] on the relevant release branch with the proper release version ([Tag][tag]).
+- [ ] Run the [CI workflow][ci-workflow] on the release branch and verified that the tagged build version exists on the [releases.crossplane.io] `build` channel. ([build][release-build]).
+- [ ] Run the [Configurations workflow][configurations-workflow] on the release branch and verified  that version exists on [registry.upbound.io] for all getting started packages.
+      - [ ] [xp/getting-started-with-aws]
+      - [ ] [xp/getting-started-with-aws-with-vpc]
+      - [ ] [xp/getting-started-with-azure]
+      - [ ] [xp/getting-started-with-gcp]
```

Comment:
These appear to be rendering weird for in the preview, does it look the same for you? I believe they may have an extra level of indentation.

Reply:
Ah yes - they are for me too. I added extra indentation to appease my markdown linter - I'll just revert that.